src/plugins/discover/public/application/angular/doc_table/components/table_row/details.html

```diff
@@ -15,28 +15,12 @@
         </div>
       </div>
     </div>
-    <div class="euiFlexItem euiFlexItem--flexGrowZero euiText euiText--small">
-      <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow">
-        <div class="euiFlexItem euiFlexItem--flexGrowZero euiText euiText--small">
-          <a
-            class="euiLink"
-            data-test-subj="docTableRowAction"
-            ng-href="{{ getContextAppHref() }}"
-            ng-if="indexPattern.isTimeBased()"
```

Comment:
I'm not seeing this logic in the src/plugins/discover/public/plugin.ts, could we include that?

Reply:
done